src/transformers/modeling_utils.py

```diff
@@ -4120,15 +4120,38 @@ def _load_pretrained_model(
         expected_keys = list(model_state_dict.keys())
         prefix = model.base_model_prefix
 
+        error_msgs = []
+
+        old_keys = []
+        new_keys = []
+        renamed_keys = {}
+
+        warning_msg = f"This model {type(model)}"
+
+        original_loaded_keys = loaded_keys.copy()
+
         def _fix_key(key):
             if "beta" in key:
                 return key.replace("beta", "bias")
             if "gamma" in key:
                 return key.replace("gamma", "weight")
-            return key
-
-        original_loaded_keys = loaded_keys
-        loaded_keys = [_fix_key(key) for key in loaded_keys]
+            return None
+
+        for i, key in enumerate(loaded_keys):
+            new_key = _fix_key(key)
```

Comment:
Rather than try and use the existing `_fix_key` logic - it would be better to rework this to: 

* Not use `copy`
* To only add the first renaming case for gamma and beta respectively

Reply:
ok , I will do it